spec/requests/custom_buttons_spec.rb

```diff
@@ -131,7 +131,7 @@
       request = {
         'action'    => 'edit',
         'resources' => [
-          { 'id' => cb.id.to_s, 'resource' => {'name' => 'updated 1', 'resource_action' => {'ae_namespace' => 'SYSTEM2'}, 'visibility' => {'roles' => ['_ALL_']}}},
+          { 'id' => cb.id.to_s, 'resource' => {'name' => 'updated 1', 'resource_action' => {'ae_namespace' => 'SYSTEM2', :ae_attributes => {"attribute" => "is present"}}, 'visibility' => {'roles' => ['_ALL_']}}},
```

Comment:
Maybe `:ae_attributes` => `'ae_attributes'` ? - as we have string keys here and we have also keys strings in api request.